portfolio/src/main/webapp/script.js

```diff
@@ -97,23 +135,24 @@ function createMap() {
 }
 
 async function getData(maxComments) {
-    if (!maxComments) maxComments = defaultMaxComments;
-    try {
-        const ret = await fetch('/data?maxComments=' + maxComments);
-        const comments = await ret.json();
-        const commentDiv = document.querySelector('#comments');
-        commentDiv.innerHTML = "";
-        comments.forEach((comment, i) =>
-            commentDiv.innerHTML += `<p>Comment ${i + 1} is ${comment}</p>`);
-    } catch (e) {
-        alert(`Hey!! There was an error: ${e?.message || e}`);
-    }
+  if (!maxComments) maxComments = defaultMaxComments;
+  try {
+      const ret = await fetch('/data?maxComments=' + maxComments);
+      const comments = await ret.json();
+      const commentDiv = document.querySelector('#comments');
+      commentDiv.innerHTML = "";
+      comments.forEach((comment, i) =>
+          commentDiv.innerHTML += `<p>Comment ${i + 1} is ${comment}</p>`);
+  } catch (e) {
+      alert(`Hey!! There was an error: ${e?.message || e}`);
+  }
 }
 
 function init() {
     composeWorld(greet)('!').split(' ').map(addHappiness).forEach(word => alert(word));
     getData();
     createMap();
+    const happinessChart = new HappinessChart('happy-regions');
 }
 
 window.onload = init;
```

Comment:
Add an empty line at the end for files. Useful for the programs that read files to know the end